content/livesync/models.textile

```diff
@@ -0,0 +1,356 @@
+
+The following diagram provides a simplified overview of the Models SDK:
+
+TODO: Diagram of flow between backend, models SDK and the user interface
+
+h2(#authenticate). Authenticate
+
+An "API key":/auth#api-keys is required for your frontend app to authenticate with Ably. API keys are used either to authenticate directly with Ably using "basic authentication":/auth/basic, or to generate tokens for untrusted clients using "token authentication":/auth/token.
+
+<aside data-type='important'>
+  <p>The code examples shown as part of the documentation use "basic authentication":/auth/basic to demonstrate features for convenience. In your own applications, basic authentication should never be used on the client-side as it exposes your Ably API key. Instead use "token authentication":/auth/token. </p>
+</aside>
+
+"Sign up":https://ably.com/sign-up to Ably to create an API key in the "dashboard":https://ably.com/dashboard or use the "Control API":/account/control-api to create an API key programmatically.
+
+API keys and tokens have a set of "capabilities":/auth/capabilities assigned to them that specify which operations, such as @subscribe@ or @publish@ can be performed on which resources. To use the Models SDK, the API key requires the following capabilities, these two capabilities are discussed further down the page, but for the time being set these two and you can review them further along:
+
+* @subscribe@ for the channels you intend to subscribe to.
+* @history@ if you intend to "sync from historical messages":#replay-from-history.
+
+h2(#install). Install
+
+The Models SDK requires a realtime client created using the "Ably JavaScript SDK":/getting-started/setup to interact with the Ably service.
+
+Install the Ably JavaScript SDK and the Models SDK from "NPM:":https://www.npmjs.com
+
+```[sh]
+npm install ably @ably-labs/models
+```
+
+Import the SDKs into your project:
+
+```[javascript]
+import ModelsClient from '@ably-labs/models';
+import { Realtime } from 'ably';
+```
+
+Instantiate a realtime client using the Ably JavaScript SDK and pass the generated client into the Models constructor:
+
+```[javascript]
+const ably = new Realtime({ key: '...' });
+const modelsClient = new ModelsClient({ ably });
+```
+
+h3(#client-options). ClientOptions
+
+The Models SDK uses a default set of @ClientOptions@. Customization of these options is not a requirement, especially if using this library for the first time.
+
+In addition to the underlying Ably realtime client, you can provide a number of other @ClientOptions@ to configure the behavior of the Models SDK:
```

Comment:
I feel like this paragraph and the one above kind of saying the same thing. Maybe we could merge them with something like: 

In addition to the underlying Ably realtime client, you can provide a number of other @ClientOptions@ to configure the default behavior of the Models SDK. Customization of these options is not a requirement, especially if using this library for the first time.